tools/openmrs_setup

```diff
@@ -71,6 +71,33 @@ mysql -uroot -p <<< "
 "
 echo 'Added MySQL user "openmrs_user" with password "openmrs".'
 
+# Running from Linux or Mac? Set the appropriate path.
+OS=`uname`
+if [[ "$OS" == 'Linux' ]]; then
+    buendia_dir='/usr/share/buendia'
+    bin_dir='/usr/bin'
+elif [[ "$OS" == 'Darwin' ]]; then
+    buendia_dir='/usr/local/opt/buendia'
+    bin_dir='/usr/local/bin'
+fi
+profiles_dir="$buendia_dir/profiles"
+
+# Create directories and copy scripts
+sudo mkdir -p $profiles_dir
+sudo cp -p tools/profile_validate "$bin_dir/buendia-profile-validate"
+sudo cp -p packages/buendia-utils/data/usr/share/buendia/utils.sh "$buendia_dir/utils.sh"
+
+# in the profile_apply script there is a hardcoded reference to utils.sh. sed will replace this a mac friendly url.
+# TODO: review this after system wide path related changes.
+if [[ "$OS" == 'Darwin' ]]; then
+    sed 's/share/local\/opt/g' tools/profile_apply > "$bin_dir/buendia-profile-apply"
```

Comment:
This looks scary... if the word `share` appears anywhere in the script it will get changed.  How about doing this logic in Python in `profile_apply`, rather than using `sed` to edit the Python code?